frontend/src/store/modules/member.ts

```diff
@@ -68,6 +68,17 @@ const actions = {
       return convert(member, data.included, rootGetters);
     });
 
+    // sort the member list
```

Comment:
If we want to maintain a stable order, maybe we should just order by createdTs?

Otherwise, if user changes the role, the changed user would be move to the top suddenly